README.md

```diff
@@ -97,18 +97,19 @@ A Linux/macOS environment is required at the moment.
 5. run `scripts/generate-proxy-api-models.sh` to generate the models defined in api_proxy.yaml and api_notifications.yaml
 6. run `scripts/generate-api-client.sh` to generate the Autorest API Client
 7. run `scripts/build.sh` to compile the Typescript files
-8. run `docker-compose up -d` to start the containers
-9. edit your `/etc/hosts` file by adding:
-
-```
-localhost    spid-testenv-identityserver
-localhost    italia-backend
-```
-
-7. wait a couple of minutes to let the IDP start (or monitor the process with `$ tail -f logs/idp/wso2carbon.log`)
-8. run `scripts/import-spid-data.sh` to configure the local IDP
-9. copy `app/.env.example` to `app/.env` and fill the variables with your values
-10. point your browser to [https://italia-backend](https://italia-backend)
+8. run `yarn generate:test-certs` to create SAML (SPID) certificates
```

Comment:
FYI, if use the markdown notation 

```markdown
1.   first list item
1.   second list item
1.   third list item
```

you don't need to reorder the numbers everytime you change the list :-)

Reply:
You are right. I did it just for matching the pre-compiled version to the expected result.